fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/ModeShapeRepositoryFactoryBean.java

```diff
@@ -151,4 +153,30 @@ private DefaultPropertiesLoader getPropertiesLoader() {
         }
         return propertiesLoader;
     }
+
+    /**
+     * Set the mode for server managed properties.  The value "strict" (default)
+     * will result in the classic behavior where certain provenance-related
+     * properties cannot be set, while "relaxed" will allow for them to be
+     * set in some circumstances.
+     *
+     * @param value the value (must be either "strict" or "relaxed")
+     */
+    public void setServerManagedPropertiesMode(final String value) {
```

Comment:
Let's leave things be for now. I have updated the following ticket to include this new System Property:
https://jira.duraspace.org/browse/FCREPO-2434